### PR TITLE
Remove capture shorcodes from #22486

### PR DESCRIPTION
**Pull request by raelga**
Replaces capture shortcodes added in https://github.com/kubernetes/website/pull/22486

/cc @sftim

### CODE REVIEW

File: content/es/docs/tasks/debug-application-cluster/debug-init-containers.md [hunk 1]
```diff
@@ -104,25 +100,21 @@ Los Init Containers que ejecutan secuencias de línea de comandos muestran los c
...
-`PodInitializing` o `Running` | El Pod ya ha terminado de ejecutar sus Init Containers.
-
-{{% /capture %}}
+| Estado                        | Significado                                                               |
+| ----------------------------- | ------------------------------------------------------------------------- |
+| `Init:N/M`                    | El Pod tiene `M` Init Containers, y por el momento se han completado `N`. |
+| `Init:Error`                  | Ha fallado la ejecución de un Init Container.                             |
+| `Init:CrashLoopBackOff`       | Un Init Container ha fallado de forma repetida.                           |
+| `Pending`                     | El Pod todavía no ha comenzado a ejecutar sus Init Containers.            |
+| `PodInitializing` o `Running` | El Pod ya ha terminado de ejecutar sus Init Containers.                   |
```
Comment (sftim) [inline content/es/docs/tasks/debug-application-cluster/debug-init-containers.md:117]: :warning: These changes are not related to capture shortcodes.

Comment (raelga) [reply content/es/docs/tasks/debug-application-cluster/debug-init-containers.md:117]: Markdown formatting on save, but I will close this PR and approve yours https://github.com/kubernetes/website/pull/22492.

**Comment by raelga** [comment]
This has already been fixed in https://github.com/kubernetes/website/pull/22492

/close
